### PR TITLE
Account import mnemonic component

### PR DESCRIPTION
**Pull request by David0817**
add import mnemonic component  # 25

### CODE REVIEW

File: src/language/en-US.json [hunk 1]
```diff
@@ -799,7 +799,7 @@
   "Input_mnemonic": "Enter a mnemonic passphrase",
   "Return_password_setting": "Return to password settings",
   "Select_public_key": "Select the imported wallets",
-  "Input_mnemonic_tips": "The mnemonic passphrase is composed of 24 randomly chosen words. After setting the words, your assets will be restored.",
+  "Input_mnemonic_tips": "Please separate the words with spaces. The number of mnemonics is 24.  The mnemonic passphrase is composed of 24 randomly chosen words. After setting the words, your assets will be restored.",
```
Comment (evias) [inline src/language/en-US.json]: `Enter the 24 words of your mnemonic passphrase separated by space.`

File: src/language/en-US.json [hunk 2]
```diff
@@ -902,7 +902,7 @@
   "Backup_Mnemonic_Phrase": "Backup Mnemonic Phrase",
   "Verify_Mnemonic_phrase": "Verify Mnemonic phrase",
   "Finish": "Finish",
-  "Type_Mnemonic_Phrase": "Type Mnemonic Phrase",
+  "Mnemonic_Phrase": "Mnemonic Phrase",
```
Comment (evias) [inline src/language/en-US.json]: `Mnemonic Passphrase`

Comment (David0817) [reply src/language/en-US.json]: thank you so much!

**Comment by evias** [comment]
> 1. it works fine when "writing down" words.

Screenshot 2020-03-16 at 8 02 23 PM

> 2. It doesn't work when "copy/paste" functionality is used

Screenshot 2020-03-16 at 8 04 07 PM

**Comment by evias** [comment]
Please, also *rebase the branch* when you work on these comments. Thanks 👍

**Comment by David0817** [comment]
@evias 
I have a question, under the current branch. the commits was rolled back always when I rebase , can you tell me why?
 thanks.

**Comment by evias** [comment]
When you rebase a branch, your commits are re-added on top of the master tip. It will **not** rollback your changes

**Comment by evias** [comment]
open a PM with me on slack to discuss this please 👍

**Pull request closed without merging**